rust/operator-binary/src/superset_controller.rs

```diff
@@ -74,6 +81,21 @@ pub enum Error {
         source: stackable_operator::error::Error,
         rolegroup: RoleGroupRef<SupersetCluster>,
     },
+    #[snafu(display("failed to build config file for {}", rolegroup))]
+    BuildRoleGroupConfigFile {
+        source: FlaskAppConfigWriterError,
+        rolegroup: RoleGroupRef<SupersetCluster>,
+    },
+    #[snafu(display("failed to build ConfigMap for {}", rolegroup))]
+    BuildRoleGroupConfig {
```

Comment:
Maybe rename it to something like `BuildRoleGroupConfigMap` so that it matches error description

Reply:
It is also called `BuildRoleGroupConfig` in all other operators (https://github.com/search?q=org%3Astackabletech+BuildRoleGroupConfig&type=code). I do not know if fixing it or being consistent is worth more.

Reply:
No opinion from my side, feel free to leave it as is